examples/client/Locomotion/src/pages/ActiveRide/index.js

```diff
@@ -313,6 +313,13 @@ const RidePage = ({ mapSettings, navigation }) => {
     },
   };
 
+  const onPressHeaderBack = () => {
+    if (currentBsPage === BS_PAGES.CONFIRM_PICKUP_TIME) {
```

Comment:
backToMap should handle this case

Reply:
you want to make backtomap a more general back function for the bottom sheet pages?